demisto_sdk/commands/lint/resources/pylint_plugins/base_checker.py

```diff
@@ -1,35 +1,34 @@
+import os
+
 from pylint.checkers import BaseChecker
 from pylint.interfaces import IAstroidChecker
 
 # You can find documentation about adding new checker here:
 # http://pylint.pycqa.org/en/latest/how_tos/custom_checkers.html#write-a-checker
 
-Msg_XSOAR_linter = {
-    'base_checker': {
-        "E9001": ("Sys.exit use is found, Please use return instead.", "sys-exit-exists",
-                  "Ensure to not use sys.exit in the code.",),
-        "E9002": ("Print is found, Please remove all prints from the code.", "print-exists",
-                  "Please remove all prints from the code.",)
-    },
-    'community_level_checker': {},
-    'partner_level_checker': {},
-    'certified_partner_level_checker': {},
-    'xsoar_level_checker': {}
+base_msg = {
+    "E9002": ("Print is found, Please remove all prints from the code.", "print-exists",
+              "Please remove all prints from the code.",),
+    "E9003": ("Sleep is found, Please remove all sleep statements from the code.", "sleep-exists",
+              "Please remove all sleep statments from the code.",),
 }
 
 
+# -------------------------------------------- Messages for all linters ------------------------------------------------
+
+
 class CustomBaseChecker(BaseChecker):
     __implements__ = IAstroidChecker
     name = "base-checker"
     priority = -1
-    msgs = Msg_XSOAR_linter.get('base_checker')
+    msgs = base_msg
```

Comment:
can you explain why this is needed?

Reply:
This is the list of messages which are relevent to this checker. every time an error is found this error is added to pylint's list of errors. as each checker inherents from Pylints BaseChecker, it has to have it.